setup.py

```diff
@@ -11,28 +11,28 @@
         "requests >= 2.22.0",
     ]
 
-with open('README.md', 'r', encoding='utf-8') as f:
+with open('README.rst', 'r', encoding='utf-8') as f:
         readme = f.read()
 
 setup(
     name="anonlink-client",
-    version='0.1.0',
+    version='0.0.1',
     description='Client side tool for clkhash and blocklib',
     long_description=readme,
     long_description_content_type='text/markdown',
```

Comment:
This should now be something for restructuredText so that it renders nicely in PyPi.

Reply:
Hi Brian, can you give more details on this one? Thanks!

Reply:
Sure the `long_description_content_type` should be the content type of our readme. So now that you've changed back to markdown it is correct without changing it as `'text/markdown'`.